package-lock.json

```diff
@@ -5095,74 +5084,88 @@
       "integrity": "sha512-K0aGNn1TjalKj+65D7ycc1//H9roAQ51GJVk5ZJQFb2teECGmzd86bYDC0aYdbRf7gtovescq4Zt6FR0tgXiHQ=="
     },
     "node_modules/@react-navigation/bottom-tabs": {
-      "version": "5.11.15",
-      "resolved": "https://registry.npmjs.org/@react-navigation/bottom-tabs/-/bottom-tabs-5.11.15.tgz",
-      "integrity": "sha512-TBY419W6aN/HZg98xbVp5Bx1HEF5sXuHR5f55W6KMI4k2AvxlwelKD1wbfvEcX2iuQT0YUiiXsACRFUSECYhkw==",
+      "version": "6.5.8",
+      "resolved": "https://registry.npmjs.org/@react-navigation/bottom-tabs/-/bottom-tabs-6.5.8.tgz",
+      "integrity": "sha512-0aa/jXea+LyBgR5NoRNWGKw0aFhjHwCkusigMRXIrCA4kINauDcAO0w0iFbZeKfaTCVAix5kK5UxDJJ2aJpevg==",
       "dependencies": {
-        "color": "^3.1.3",
-        "react-native-iphone-x-helper": "^1.3.0"
+        "@react-navigation/elements": "^1.3.18",
+        "color": "^4.2.3",
```

Comment:
We need to take care with color packages. See this: https://www.bleepingcomputer.com/news/security/dev-corrupts-npm-libs-colors-and-faker-breaking-thousands-of-apps/

Today it is ok: https://security.snyk.io/package/npm/color

However we should take take.

Reply:
Good consideration. This is a dependency of the `navigation-bottom-tab`, a widely used dependency that we currently really need.